src/main/java/com/okta/tools/helpers/SessionHelper.java

```diff
@@ -81,35 +88,20 @@ public void updateCurrentSession(Instant expiryInstant, String profileName) thro
     }
 
     public Optional<Profile> getFromMultipleProfiles() throws IOException {
-        return getMultipleProfile().getProfile(environment.oktaProfile, getMultipleProfilesPath());
+        return FileHelper.readingPath(FileHelper.getOktaDirectory().resolve("profiles"), reader ->
+            new MultipleProfile(reader).getProfile(environment.oktaProfile)
+        );
     }
 
     public void addOrUpdateProfile(Instant start) throws IOException {
-        MultipleProfile multipleProfile = getMultipleProfile();
-        multipleProfile.addOrUpdateProfile(environment.oktaProfile, environment.awsRoleToAssume, start);
-
-        try (final FileWriter fileWriter = FileHelper.getWriter(FileHelper.getOktaDirectory(), "profiles")) {
-            multipleProfile.save(fileWriter);
-        }
+        FileHelper.usingPath(FileHelper.getOktaDirectory().resolve("profiles"), (reader, writer) -> {
+            MultipleProfile multipleProfile = new MultipleProfile(reader);
+            multipleProfile.addOrUpdateProfile(environment.oktaProfile, environment.awsRoleToAssume, start);
+            multipleProfile.save(writer);
+        });
     }
 
     public boolean sessionIsActive(Instant startInstant, Session session) {
         return startInstant.isBefore(session.expiry);
     }
-
-    private void logoutMultipleAccounts(String profileName) throws IOException {
-        cookieHelper.clearCookies();
-
-        getMultipleProfile().deleteProfile(getMultipleProfilesPath().toString(), profileName);
-    }
-
-    private Path getMultipleProfilesPath() throws IOException {
-        return FileHelper.getFilePath(FileHelper.getOktaDirectory(), "profiles");
-    }
-
-    private MultipleProfile getMultipleProfile() throws IOException {
-        Reader reader = FileHelper.getReader(FileHelper.getOktaDirectory(), "profiles");
-
-        return new MultipleProfile(reader);
-    }
```

Comment:
:art: I reordered the methods here to be in step-down order (Clean Code). Sorry for the diff carnage.